src/components/reservation/ReservationModalBody.tsx

```diff
@@ -134,7 +141,10 @@ const ReservationModalBody = ({
               <div className="reservation-modal-tag">
                 {materialTypes[0].specific}
               </div>
-              <h2 className="text-header-h2 mt-22 mb-8">{titles.main[0]}</h2>
+              <h2 className="text-header-h2 mt-22 mb-8">
+                {titles.main[0]}{" "}
```

Comment:
You can also destruct it in the function definition:
```
mainManifestation: { pid, materialTypes, titles: { [main: mainTitle] }, edition }
```
Haven't tested so there could be mistakes.
If you are not fond of the destructing I think at least you should create a variable or a function for it since you refer to it twice in this component. It would make sense to create a funcvtion for it since the main title is also used in other places of the codebase.